models/_sector/tokens/tokens_transfers.sql

```diff
@@ -0,0 +1,75 @@
+{{config(
+    schema = 'tokens',
+    alias = 'transfers',
+    partition_by = ['token_standard', 'block_date', 'blockchain'],
```

Comment:
I think the order here matters and block_date should be last. I also think maybe we should drop `blockchain` here. If you are querying this table I'm assuming you'd want all or most chains.

Reply:
interesting on the order, didn't realize that. i can reorder.

i think i'd push back on removing `blockchain`, as i think most queries will either filter or group by on `blockchain` on this crosschain spell. isn't that the best use case for adding a partition?

Reply:
Yeah maybe you are right, we can keep it then